recipe/meta.yaml

```diff
@@ -10,12 +10,15 @@ source:
   sha256: 3e2e3a06580c5f190df843cdb90ea28d61099cf4924334d5297a995de68e4673
 
 build:
-  number: 0
+  number: 1
```

Comment:
I'm not sure we need to bump the build number

Reply:
I've seen it once that not bumping resulted in two incompatible versions with differing dependencies for another platform (because of the rerender) and since always bump just to be sure. But happy to leave it at 0 if you don't think it's an issue.